domain-services/fiscal-event-service/pom.xml

```diff
@@ -76,26 +76,25 @@
       <artifactId>validation-api</artifactId>
     </dependency>
 
-    <dependency>
-      <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot-starter-data-mongodb</artifactId>
-    </dependency>
-
-    <dependency>
-      <groupId>com.github.cloudyrock.mongock</groupId>
-      <artifactId>mongock-spring-v5</artifactId>
+   <dependency>
+      <groupId>org.postgresql</groupId>
+      <artifactId>postgresql</artifactId>
```

Comment:
Mention the version number. In most of the DIGIT services, the version number is mentioned in the `properties` section near the top.

Reply:
minor change, to be taken up as part of code cleanup.